internal/k8s/k8s.go

```diff
@@ -189,12 +190,12 @@ func New(cfg *Config) (*Client, error) {
 
 				c.syncFuncs = append(c.syncFuncs, c.epInformer.HasSynced)
 			} else {
-				c.logger.Log("op", "New", "msg", "using endpoint slices")
+				level.Info(c.logger).Log("op", "New", "msg", "using endpoint slices")
```

Comment:
I'd go with debug here

Reply:
Endpoint slices are still new in MetalLB so it's a good information to have in bug report for now IMO.

Reply:
(and it's just 1 line)

Reply:
I guess so, but it's an implementation detail that shouldn't matter unless something is broken (like many other debug logs).

Like you said, it's just 1 line so I don't feel that strongly ...

Reply:
Should I do this change? Or you can do it directly.